srcpkgs/oilshell/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'oilshell'
+pkgname=oilshell
+version=0.6.0
+revision=1
+wrksrc=oil-${version}
+build_style=configure
+configure_args="--prefix=/usr $(vopt_with readline)"
+makedepends="$(vopt_if readline readline-devel)"
+short_desc="Oil, a new unix shell"
+maintainer="Caleb JA <skywise@tutanota.de>"
+license="Apache-2.0"
+homepage="https://www.oilshell.org"
+distfiles="${homepage}/download/oil-${version}.tar.xz"
+checksum=141e3a5c1997a696d65747966978aaa38921d77e303aad9e77b4ab4aedab84b7
+register_shell="/usr/bin/osh"
+nocross="oilshell is nocross"
+nostrip=yes
+
+build_options="readline"
+build_options_default="readline"
+
+pre_build() {
+	# Remove the strip wrapper
+	rm -f "${XBPS_WRAPPERDIR}"/strip
+}
+
+do_install() {
+	# Remove the strip wrapper
+	rm -f "${XBPS_WRAPPERDIR}"/strip
+
+	# Install binaries
+	vbin _bin/oil.ovm
+	# Symlink osh, oshc to oil.ovm.
+	cd "${DESTDIR}"/usr/bin
+	ln -s oil.ovm osh
+	ln -s oil.ovm oshc
+}
+
+post_install() {
+	vlicense LICENSE.txt
+}
```

Comment:
please add a newline at the end of the file

Reply:
Added a newline